test/dshell/extra-files/printenv.d

```diff
@@ -1,7 +1,15 @@
 import std.array, std.stdio, std.process, std.algorithm;
+
+auto filterEnv(T)(T env)
+{
+    // Windows shell output is not in UTF-8
+    // https://github.com/dlang/phobos/pull/7342#issuecomment-571154708
+    return env.filter!(e => e.key.among("BUILD_SOURCEVERSIONAUTHOR"));
```

Comment:
Maybe add a bang here, so it shows everything except `BUILD_SOURCEVERSIONAUTHOR`, rather than only that?